src/box/alter.cc

```diff
@@ -3404,10 +3405,11 @@ func_def_new_from_tuple(struct tuple *tuple)
 			return NULL;
 		uint32_t argc = mp_decode_array(&param_list);
 		for (uint32_t i = 0; i < argc; i++) {
-			 if (mp_typeof(*param_list) != MP_STR) {
+			enum mp_type actual_type = mp_typeof(*param_list);
+			 if (actual_type != MP_STR) {
```

Comment:
Extra whitespace before `if` (I know it was there before, but still you can delete it since you change this line anyway).